frontend/src/component/Login/SignIn.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { withRouter, Link } from 'react-router-dom'
+import { translate } from 'react-i18next'
+import InputGroupText from '../common/Input/InputGroupText.jsx'
+import { IconButton, PAGE } from 'tracim_frontend_lib'
+
+const SignIn = props => {
+  return (
+    <div className='loginpage__main__wrapper'>
+      <div className='loginpage__main__header'>
+        <h1 className='loginpage__main__header__title'>{props.t('Sign in')}</h1>
+        {props.system.config.user__self_registration__enabled && (
+          <IconButton
+            customClass='loginpage__main__header__createButton'
+            text={props.t('Create account')}
+            icon='fas fa-user-plus'
+            onClick={props.onClickCreateAccount}
+            intent='secondary'
+          />
+        )}
+      </div>
+      <form onSubmit={props.onClickSubmit} noValidate className='loginpage__main__form'>
```

Comment:
I wonder if the re-use of the same class name might be confusing. Maybe not. If you don't think so, feel free to resolve the conversation.

Reply:
I don't think it's confusing, but it's maybe a good practice to do that. I don't know, I don't have a strong opinion about it.